spec/models/vote_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe Vote, type: :model do
+  describe "#to_s" do
+    it "returns the vote's type and id" do
+      user = User.create(username: "Sample_Username", email: "secret_email", password: "secret_password")
+      parent = Link.create(title: "Sample Link", url: "Sample URL", user: user)
```

Comment:
These two lines are totally fine, but if I were writing this test myself, I'd utilize `let`, `subject`, and multi-line some of these long assignments:
```
Rspec.describe Vote, type: :model do
  let!(:user) do
    User.create(
      username: "Sample_Username",
      email:    "secret_email",
      password: "secret_password"
    )
  end

  let!(:parent) do
    Link.create(
      title: "Sample Link",
      url:   "Sample URL",
      user:  user
    )
  end

  subject { Vote.create(votable: parent) }

  describe "#to_s" do
    it "returns the vote's type and id" do
      expect(subject.to_s).to eql "Link: #{parent.id}"
    end
  end
end
```